cardano-node/src/Cardano/Node/Configuration/POM.hs

```diff
@@ -284,7 +283,7 @@ makeNodeConfiguration pnc = do
   shutdownIPC <- lastToEither "Missing ShutdownIPC" $ pncShutdownIPC pnc
   shutdownOnSlotSynced <- lastToEither "Missing ShutdownOnSlotSynced" $ pncShutdownOnSlotSynced pnc
   protocolConfig <- lastToEither "Missing ProtocolConfig" $ pncProtocolConfig pnc
-  viewMode <- lastToEither "Missing ViewMode" $ pncViewMode pnc
+  _viewMode <- lastToEither "Missing ViewMode" $ pncViewMode pnc
```

Comment:
We do not need this at all at this level. Indeed we can remove `pncViewMode` too.

If we really want to warn, we can do that when we read the json, but it cannot be a parse failure.

Reply:
Yes, @dcoutts, it's much simpler for me just remove it completely.